airbyte-notification/src/main/java/io/airbyte/notification/CustomeriolNotificationClient.java

```diff
@@ -29,12 +30,15 @@ public class CustomeriolNotificationClient extends NotificationClient {
   // - receiver email
   // - customer.io identifier email
   // - customer.io TRANSACTION_MESSAGE_ID
-  private static final String SENDER_EMAIL = "Airbyte Notification <no-reply@airbyte.io>";
-  private static final String TRANSACTION_MESSAGE_ID = "6";
+
+  // DEFAULT_TRANSACTION_MESSAGE_ID is currently unused but the template can be used for any generic
+  // messaging.
+  // private static final String DEFAULT_TRANSACTION_MESSAGE_ID = "6";
+  private static final String AUTO_DISABLE_TRANSACTION_MESSAGE_ID = "7";
+  private static final String AUTO_DISABLE_WARNING_TRANSACTION_MESSAGE_ID = "8";
```

Comment:
sorry i'm not that familiar with this part of the codebase.  What do these numbers actually mean/how are they used?

Reply:
they relate to customer.io email templates. I essentially moved the messaging specifics from the json files into there, since some of the messaging were cloud-specific, such as reaching out to cloud support for help, etc.

Reply:
ah ok, thanks!

Reply:
I left in a default template `DEFAULT_TRANSACTION_MESSAGE_ID = "6"` where anyone from the team (using airbyte's own customer.io api key) can use and add their own messaging to.

Reply:
hmm, would it help to update the comments a bit to mention all of this?